src/cio_websocket.c

```diff
@@ -116,102 +140,131 @@ static void send_frame(struct cio_websocket *ws, struct cio_write_buffer *payloa
 		first_len |= WS_MASK_SET;
 		uint8_t mask[4];
 		cio_random_get_bytes(mask, sizeof(mask));
-		memcpy(&ws->send_header[header_index], &mask, sizeof(mask));
+		memcpy(&job->send_header[header_index], &mask, sizeof(mask));
 		header_index += sizeof(mask);
 		// TODO: cio_websocket_mask(payload, length, mask );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this block of commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3ACommentedCode)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS1135)